lib/query_components/visibility_filter.rb

```diff
@@ -0,0 +1,13 @@
+require 'elasticsearch/queries'
+
+module QueryComponents
+  class VisibilityFilter < BaseComponent
+    include Elasticsearch::Queries
+
+    def rejects_payload
+      return [] if search_params.debug[:withdrawn]
+
+      [term_filter("is_withdrawn", true)]
```

Comment:
Would it be clearer to inline what `term_filter` does here? `[{ term: { is_withdrawn: true } }]` isn't that much longer and prevents some indirection.